common/dbutils/bucket.go

```diff
@@ -96,8 +98,10 @@ var (
 	StorageChangeSetBucket = "SCS"
 
 	// some_prefix_of(hash_of_address_of_account) => hash_of_subtrie
-	IntermediateTrieHashBucket     = "iTh2"
-	IntermediateTrieHashBucketOld1 = "iTh"
+	IntermediateHashOfAccountBucket = "ih_account"
```

Comment:
Is "intermediate" still correct?

Reply:
What do you mean? 
This bucket storing prefixes and hashes of branch nodes in merkle trie (which are shorter than account prefix) - "intermediate" it's just poor naming for this.